modules/smart-agent_kubernetes-common/conf/01-hpa-scale-capacity.yaml

```diff
@@ -0,0 +1,15 @@
+module: kubernetes
+name: "hpa reached max"
+tip: "hpa reached max for too long, the maximum number of desired Pods has been hit"
+signals:
+  max:
+    metric: kubernetes.hpa.spec.max_replicas
+  current:
+    metric: kubernetes.hpa.status.current_replicas
+  signal:
+    formula: (max-current)
+rules:
+  major:
+    threshold: 0
+    comparator: "!="
```

Comment:
erf, my bad, indeed my mistake was reverse thinking 
i want to raise an alert if current replicas number is on max replicas for too long, so `==` instead of `!=` (if difference is 0 for too long)